test/test_shared.cpp

```diff
@@ -89,6 +91,32 @@ using unit_test::TestContext;
 // check-testcase` (or one of its friends) from the command line.
 
 
+#ifdef _WIN32
+namespace {
+    // This does NOT work like on POSIX: The child will begin execution from the unit
+    // test entry point, not from where fork() took place.
+    DWORD winfork(std::string unit_test_name)
+    {
+        if (getenv("REALM_FORKED"))
+            return GetCurrentProcessId();
+
+        remove("winfork.bat");
+        std::ofstream myfile("winfork.bat");
```

Comment:
I don't know why my reply to ironage was placed outside this thread. But he saw it